### ISSUE TITLE
[DataCap Application] Distributed Archives for Neurophysiology Data Integration

### ISSUE THREAD

**Issue by kiw-max** [LendMi-Finance/LendMi-Storage#19]
### Version

1

### DataCap Applicant

Distributed Archives for Neurophysiology Data Integration

### Project ID

DANDI-001

### Data Owner Name

Distributed Archives for Neurophysiology Data Integration

### Data Owner Country/Region

Afghanistan

### Data Owner Industry

Life Science / Healthcare

### Website

https://www.dandiarchive.org/

### Social Media Handle

https://www.dandiarchive.org/

### Social Media Type

Twitter

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

10PiB

### Expected size of single dataset (one copy)

2PiB

### Number of replicas to store

5

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f1ne47l72r2hgfi7q3no6s7cnf6alr6vxp2x5k5zq

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
We are Filecoin investors, we have a technical team, and we hope to help SP complete digital transformation and store valuable data.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
DANDI is a public archive of neurophysiology datasets, including raw and processed data, and associated software containers. Datasets are shared according to a Creative Commons CC0 or CC-BY licenses. The data archive provides a broad range of cellular neurophysiology data. This includes electrode and optical recordings, and associated imaging data using a set of community standards: NWB:N - NWB:Neurophysiology, BIDS - Brain Imaging Data Structure, and NIDM - Neuro Imaging Data Model.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

Singapore

### Please share a sample of the data

```text
https://registry.opendata.aws/dandiarchive/
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Yearly

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, Africa, North America, South America, Europe

### How will you be distributing your data to storage providers

Cloud storage (i.e. S3), HTTP or FTP server, IPFS, Shipping hard drives, Lotus built-in data transfer, Venus built-in data transfer

### How did you find your storage providers

Slack, Filmine, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f02852273 London
f02984331 Singapore
f02883857 Singapore
f02973061 Moscow
f02889193 Vietnam
```

### How do you plan to make deals to your storage providers

Boost client, Lotus client, Singularity

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by LendMi** [comment]
Hi, please answer the following questions:

1) Can you confirm that all your SPs support Spark retrieval?
2) It looks like this is a public data set. Do you have any experience downloading and storing it?
3) How do you process the data?

**Comment by kiw-max** [comment]
Yes, all SPs support Spark retrieval. we will help them improve their technical solutions and increase their support for Spark. This is a public dataset. We started downloading the dataset a month ago. We have sufficient experience and have helped many SPs download and store data. We have developed a data system that has been running for more than a year. It is jointly developed and maintained by many technicians. We use our own system to process data. Thank you.

**Comment by LendMi** [comment]
Can you provide more data samples? Where will you start downloading the data?

**Comment by kiw-max** [comment]
We are already preparing the data. Could you approve it for us? 
<img width="830" alt="WechatIMG842" src="https://github.com/user-attachments/assets/290e61c4-0c22-4c21-9544-59bee6caa5ae">
<img width="1413" alt="WechatIMG834" src="https://github.com/user-attachments/assets/55409b0f-3c0c-483b-9d47-6411a633e3a5">

**Comment by kiw-max** [comment]
checker:manualTrigger

**Comment by kiw-max** [comment]
We have used up, can we get the next round please?

**Comment by Rongze92** [comment]
There is a delay in the first round of bot. 
I need to see the bot data in the next round.

**Comment by kiw-max** [comment]
checker:manualTrigger

**Comment by Rongze92** [comment]
checker:manualTrigger

**Comment by kiw-max** [comment]
had been used up. Can we see more updates here? Can you give more DC in each round? For example, give us 1P or 2P?

**Comment by kiw-max** [comment]
checker:manualTrigger

**Comment by kiw-max** [comment]
DC had been used up. Can we see more updates here? Can you give more DC in each round? For example, give us 1P or 2P?

**Comment by Rongze92** [comment]
Spark's retrieval rate needs to be increased to 75%. Due to the limited remaining quota, we can only support a small amount. We hope to see more improvements.

**Comment by kiw-max** [comment]
Thanks

**Comment by kiw-max** [comment]
checker:manualTrigger

**Comment by kiw-max** [comment]
checker:manualTrigger

**Comment by kiw-max** [comment]
We are actively working to improve

**Comment by Rongze92** [comment]
Whether duplicate data has been resolved.

**Comment by Rongze92** [comment]
checker:manualTrigger

**Comment by kiw-max** [comment]
checker:manualTrigger

**Comment by kiw-max** [comment]
If you can assign us the next round, it will be solved soon.

**Comment by LendMi** [comment]
checker:manualTrigger
